python/ray/util/accelerators/accelerators.py

```diff
@@ -13,5 +13,8 @@
 GOOGLE_TPU_V3 = "TPU-V3"
 GOOGLE_TPU_V4 = "TPU-V4"
 
-# Deprecated as A100 is not part of the "TESLA" suite.
-NVIDIA_TESLA_A100 = "A100"
+# Use these instead of NVIDIA_A100 if you need a specific accelerator size. Note that
+# these labels are not auto-added to nodes, you'll have to add them manually in
```

Comment:
If we allow users to manually add them, then they need to understand `accelerator_type:xxx` custom resource which is currently an implementation detail. Ideally we don't want to expose it so that we can easily switch to labels later.

Reply:
I'm just going to merge this since standardizing the enum is important, separately from actually tagging the nodes properly.